source/v2/greenlanes-openapi.yaml

```diff
@@ -354,8 +365,8 @@ components:
           enum: [ 1, 2, 3 ]
           description: The category applicable unless exemption criteria are met
         theme:
-          type: string
-          description: The Green Lanes theme relevant for this Category Assessment
+          description: The Green Lanes Theme used as the basis for this possible Category Assessment
```

Comment:
Should this be applicable Category Assessment, instead of possible Category Assessment